packages/ag-charts-community/src/chart/axis/cartesianAxis.ts

```diff
@@ -124,6 +145,13 @@ export abstract class CartesianAxis<S extends Scale<D, number, any> = Scale<any,
         return new CartesianAxisLabel();
     }
 
+    protected getTransformBox(bbox: BBox) {
```

Comment:
I'd prefer if we could avoid direct matrix calculations, is there a simpler way to achieve this?

Reply:
It's just moved from axis - I'm going bit-by-bit with this, so it might get removed in the end